### PR TITLE
MINT-15 Render widget even when price is less than min price

### PR DESCRIPTION
**Pull request by gopida**

### CODE REVIEW

**Review by aholdahl — CHANGES_REQUESTED** [review]
Don't forget to update the widget-options.md entry!

File: widget-options.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ As with targetXPath, prepend IDs with '#', classes with '.', and append tag name
 
 `minPrice` (optional)
 
-**Purpose**: Minimum price in cents for which the widget should be rendered. If the price at `targetXPath` is lower than this number, the widget will not render on the page.<br/>
+**Purpose**: Minimum price in cents for which the widget should be rendered. If the price at `targetXPath` is lower than this number, the widget will render with a note stating the minimum eligible price required.<br/>
```
Comment (aholdahl) [inline widget-options.md:145]: Purpose and Additional Details still need further updates.

Minimum price in cents for which Sezzle can be selected at checkout.

Remove `While Sezzle is not advertised on these items`

**Review by aholdahl — CHANGES_REQUESTED** [review]

File: widget-options.md (hunk 1 above)
Comment (aholdahl) [reply widget-options.md:145]: Oh, also add this change to the upcoming version in CHANGELOG.md

Comment (gopida) [reply widget-options.md]: Ahh... I keep forgetting that. Thank you. Addressed both.

**Review by aholdahl — APPROVED** [review]

**Pull request merged**